index.html

```diff
@@ -0,0 +1,234 @@
+          </li>
+        </ul>
+      </nav>
+      <ul class="humb mobile-menu mobile-menu-toggle pos-f-t">
+        <li class="hum"><i class="fas fa-bars"></i></li>
+      </ul>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <ul class="menu">
+          <li>
+            <a href="#">
+              <img
+                id="imag-log"
+                src="./src-image/logo-image.png"
+                alt="logo image"
+              />
+            </a>
+          </li>
+          <li>
+            <a
+              class="navbar-brand"
+              id="about"
+              href="about.html"
+              rel="noreferrer noopener"
+              >About</a
+            >
+          </li>
+          <li>
+            <a class="navbar-brand" href="#" rel="noreferrer noopener"
+              >Program</a
+            >
+          </li>
+          <li>
+            <a class="navbar-brand" href="#" rel="noreferrer noopener">Join</a>
+          </li>
+          <li>
+            <a class="navbar-brand" href="#" rel="noreferrer noopener"
+              >Sponsor</a
+            >
+          </li>
+          <li>
+            <a class="navbar-brand" href="#" rel="noreferrer noopener">News</a>
+          </li>
+          <li>
+            <a class="navbar-brand" id="comp" href="#" rel="noreferrer noopener"
+              >CC Campaign</a
+            >
+          </li>
+        </ul>
+      </nav>
+      <div class="menu-overlay hide-menu">
+        <ul class="mobile-menu-toggle menu-close">
+          <li><i class="fas fa-times" id="cross"></i></li>
+        </ul>
+        <ul class="mobile-menu-items mobile-menu-toggle">
+          <li><a href="about.html" rel="noreferrer noopener">About</a></li>
+          <li><a href="#" rel="noreferrer noopener">Program</a></li>
+          <li><a href="#" rel="noreferrer noopener">Join</a></li>
+          <li><a href="#" rel="noreferrer noopener">Sponsor</a></li>
+          <li><a href="#" rel="noreferrer noopener">News</a></li>
+          <li><a href="#" rel="noreferrer noopener">CC Campaign</a></li>
+        </ul>
+      </div>
+    </header>
+    <section class="home col-12">
+      <video autoplay="" muted="" loop="" id="myVideo">
+        <source
+          src="./vedio-background/Silky_Blue_720P_Motion_Background_Loop.mp4"
+          type="video/mp4"
+        />
+      </video>
+      <h4 class="welcome h4">" Welcome for Online Course "</h4>
+      <p class="det">
+        Activities and events for the new online training courses 2022
+      </p>
+      <p class="paragh">
+        The world has changed and education has become easy to obtain, so do not
+        worry, you will start a fantastic journey from which you will gain
+        knowledge and practical experience 2022
+      </p>
+      <p class="date-p">2022.01.01 (SAT) ~ 10(MON)</p>
+      <p class="address">@Haram Giza Egypt</p>
+    </section>
+    <section class="main-program col-12">
+      <h1 class="program-ma h1">
```

Comment:
- [ ] Why don't you use a dark background image for this section. This is to ensure that your design looks almost identical to the design template. You can get a dark image on [pexels.com](https://www.pexels.com/search/computers/?color=black)

- [ ] Let's give proper spacing between the main program and the boxes.

Your Implementation | Cindy's Design
:---:|:---:
![](https://i.imgur.com/2zOIaaP.png) | ![](https://i.imgur.com/u6EpbGV.png)